### ISSUE TITLE
Incorrect MIME type for fonts

### ISSUE THREAD

**Issue by pgundlach** [bmaupin/go-epub#13]
I include an otf font into an epub. Running epubcheck gives me a warning that the epub references the font by `application/x-font-otf`. I believe it should be `application/font-sfnt` (see again https://www.w3.org/publishing/epub3/epub-spec.html#sec-cmt-supported)

I get the same warning with ttf.

BTW: thanks for the great project and the quick fix yesterday!

**Comment by bmaupin** [comment]
Good catch, it definitely looks incorrect.

I'm wondering if we should go with `application/vnd.ms-opentype` for otf fonts to maintain backwards compatibility with the 3.0.1 spec:

http://www.idpf.org/epub/301/spec/epub-publications.html#sec-core-media-types

What do you think?

Along those lines, it might be best to use `application/font-woff` for woff fonts. Although to be honest I'm not sure how much it matters.

TTF and woff2 don't seem to be part of the 3.0.1 spec so I'm guessing we could just use whatever's in the 3.2 spec.

Also, I'm having trouble reproducing this warning with epubcheck. Could you please attach the EPUB that's giving you the warning?

Thanks!

**Comment by bmaupin** [comment]
Apparently I had to actually include the font in the css to get the message. It's `INFO` and not a warning but I think it merits worth fixing anyway:

```
INFO(CSS-007): /home/bmaupin/Desktop/basic-v3plus2.epub/EPUB/css/epub.css(11,3): Font-face reference EPUB/fonts/redacted-script-regular.ttf refers to non-standard font type application/x-font-ttf.
```

**Comment by bmaupin** [comment]
I see a comment here mentioning backward compatibility is unknown: https://github.com/w3c/epubcheck/issues/339#issuecomment-332779105

So for now I'll pick the MIME types that are in both the 3.0.1 and 3.2 specifications, and if need be we can change it later.

Thanks!

**Comment by bmaupin** [comment]
Fixed with v0.5.3. Thanks!

**Comment by pgundlach** [comment]
Thank you very much! Awesome work!
